docs/service-extensions.md

```diff
@@ -46,7 +46,7 @@ these sections would look like in the configuration file:
 
 ```yaml
 
-# Example of the extensions provided in OTelSvc core. The list below
+# Example of the extensions available OOB with the Collector. The list below
```

Comment:
What does OOB stand for?

Reply:
Out of the box I guess?

Reply:
The idea was really to to be out of the box, will avoid the acronym.